Sources/Request/Helpers/MediaType.swift

```diff
@@ -9,30 +9,55 @@
 import Foundation
 
 /// A MIME type to be used with the `Accept` and `Content-Type` headers.
-public enum MediaType: String {
-    case json = "application/json"
-    case xml = "application/xml"
+public enum MediaType: CustomStringConvertible {
```

Comment:
I'm sorry, in my previous review I suggested this, but I meant to say `ExpressibleByStringLiteral`. Then we can keep the type safety throughout and have the convenience of a string literal.

```suggestion
public enum MediaType: String, ExpressibleByStringLiteral {
```
You'll need to add an initializer and `rawValue` I think.